tests/01_unit/pipelines/test_logic.py

```diff
@@ -27,7 +27,7 @@
 # chains the operations that are unit-tested.
 
 
-@pytest.fixture(params=["mono", "ckd"])
+@pytest.fixture(params=["mono", "ckd", "mono_double_polarized"])
```

Comment:
I'd prefer to keep consistency with the used aliases, and also to cover the `ckd_polarized` mode
```suggestion
@pytest.fixture(params=["mono", "ckd", "mono_polarized", "ckd_polarized"])
```